src/core/core.defaults.js

```diff
@@ -17,8 +17,7 @@ defaults._set('global', {
 	defaultFontFamily: "'Helvetica Neue', 'Helvetica', 'Arial', sans-serif",
 	defaultFontSize: 12,
 	defaultFontStyle: 'normal',
-	defaultLineHeight: 1.2,
-	showLines: true
```

Comment:
Should we consider this removal as breaking?

Reply:
At least `polarArea` and `radar` controllers should be updated with a default.

Reply:
Those controllers do not use the `showLines` option